internal/cli/version.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"runtime/debug"
+	"strings"
+	"time"
+)
+
+// Version prints the version information for the binary.
```

Comment:
Perhaps "returns" rather than "prints"? It might be worth mentioning that while it's derived from machine readable sources, the returned value is meant to be *human* readable rather than anything that's then parsed further. (Assuming I've understood that correctly.)